elements/pfe-readtime/src/pfe-readtime.js

```diff
@@ -0,0 +1,214 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+class PfeReadtime extends PFElement {
+  static get tag() {
+    return "pfe-readtime";
+  }
+
+  static get meta() {
+    return {
+      title: "Readtime",
+      description:
+        "This element will collect a word count on a given section and calculate the readtime based on that count."
+    };
+  }
+
+  get templateUrl() {
+    return "pfe-readtime.html";
+  }
+
+  get styleUrl() {
```

Comment:
Since you don't have styles, I believe you can delete this whole section and it should be fine.